_documentation/user/flags/seer.md

```diff
@@ -5,4 +5,14 @@ type: Good
 summary: See stealthed, cloaked and masquerading tanks as normal.
 ---
 
-...
+The Seer (SE) flag allows the player to see the tanks holding the [Stealth (ST)](../stealth/), [Cloaking (CL)](../cloaking/), or [Masquerade (MQ)](../masquerade/) flags as normal.
+
+Its power is therefore to nullify the effects of those three flags for the holder, rather than bringing out a unique power-up itself.
+
+## Tactics
+
+As the flag disables the effect of three other specific flags for the holder, the usage of Seer is highly situational.
```

Comment:
I'd avoid specifying a hard limit since updating this value (if ever) could be hard to remember